tests/users.js

```diff
@@ -0,0 +1,46 @@
+import test from 'ava';
+import users from '../lib/users';
+
+
+const usersContentObj = {
+  'name': 'Users',
+  'description': 'Test users description',
+  'id': 'users',
+  'attributes': [
+    {
+      'type': 'email',
+      'id': 'email',
+      'name': 'Email',
+      'description': 'Your email is your username',
+    },
+  ],
+};
+
+test('Exports exist', t => {
+  t.is(typeof users, 'object', 'Users exports a function');
+
+  t.is(typeof users.userContentType, 'function', 'Submodule `userContentType` exists and is a function');
+  t.is(typeof users.userRoutes, 'function', 'Submodule `userRoutes` exists and is a function');
+  t.is(typeof users.checkUserTable, 'function', 'Submodule `checkUserTable` exists and is a function');
+});
+
+test('Users, with config, merged with correct param', t => {
```

Comment:
Don't test your dependencies. There should be tests inside of `content-types` to test this functionality, you don't need to.